crates/edr_napi_core/src/logger.rs

```diff
@@ -0,0 +1,1193 @@
+use core::fmt::Debug;
```

Comment:
File moved from `edr_napi`, with a modification to the config. This removes the dependency on `napi` types.